CHANGELOG.md

```diff
@@ -1,5 +1,38 @@
 Changes
 =======
+
+# 1.0.0 / 01-07-2020
+### Details
+https://github.com/jenkinsci/datadog-plugin/compare/datadog-0.7.1...datadog-1.0.0
+
+### Changes
+[BACKWARDS INCOMPATIBILITY NTOES]
+* Instead of sending `null` as default value for some tags we now send `unknown`
+* Event messages got changed.
+* Node tag is set by default
+
+[BUGFIX] Fix rate submission. See [179](https://github.com/DataDog/jenkins-datadog-plugin/pull/179) #TODO Confirm if the issue fixed in this PR was in the last release or not
+
+[IMPROVEMENT] Add note about how to send data to EU [154](ttps://github.com/DataDog/jenkins-datadog-plugin/pull/134)
```

Comment:
This link doesn't work. Even with `h`. PR 134 doesn't exist.

Reply:
Also based on the description. Wouldn't it be a no changelog?

Reply:
Whoops, good catch. Its actually 154... 
It ships a change to one of the config settings help text, so I think it warrants a changelog